frontend/javascripts/viewer/geometries/plane.ts

```diff
@@ -145,45 +153,46 @@ class Plane {
     }
     this.lastScaleFactors[0] = xFactor;
     this.lastScaleFactors[1] = yFactor;
-
-    const scaleVec = new THREE.Vector3().multiplyVectors(
-      new THREE.Vector3(xFactor, yFactor, 1),
-      this.baseScaleVector,
-    );
-    this.plane.scale.copy(scaleVec);
-    this.TDViewBorders.scale.copy(scaleVec);
-    this.crosshair[0].scale.copy(scaleVec);
-    this.crosshair[1].scale.copy(scaleVec);
+    // Account for the dataset scale to match one world space coordinate to one dataset scale unit.
+    const scaleVector: Vector3 = [
+      xFactor * this.datasetScaleFactor[0],
+      yFactor * this.datasetScaleFactor[1],
+      1 * this.datasetScaleFactor[2],
+    ];
+    this.getMeshes().map((mesh) => mesh.scale.set(...scaleVector));
   }
 
+  setBaseRotation = (rotVec: THREE.Euler): void => {
+    this.baseRotation.copy(rotVec);
```

Comment:
please explain somewher what the difference between base rotation and the other rotation is.